modules/vba_documents/app/workers/vba_documents/slack_notifier.rb

```diff
@@ -42,55 +41,55 @@ def fetch_settings
 
     def daily_notification
       hour = Time.now.utc.hour + Time.zone_offset('EST') / (60 * 60)
+      results = ''
+
       if hour.eql?(@daily_notification_hour)
-        text = "Daily Status (worst offenders over past week):\n"
         statuses = UploadSubmission::IN_FLIGHT_STATUSES + ['uploaded'] - ['success']
         statuses.each do |status|
           model = UploadSubmission.aged_processing(0, :days, status).where('created_at > ?', 7.days.ago).first
           next unless model
 
           start_time = model.metadata['status'][status]['start']
           duration = distance_of_time_in_words(Time.now.to_i - start_time)
-          text += "\tStatus \'#{status}\' for #{duration}\n"
+          results += "\n\tStatus \'#{status}\' for #{duration}"
         end
-        resp = send_to_slack(text, @default_alert_url)
+
+        notify_slack('Daily Status (worst offenders over past week)', results)
+        true
```

Comment:
Returning `true` in several of these methods because the `perform` method logs the result of each "report" run. I can see many opportunities for refactoring in this class, but since we are likely going to be touching these alerts again soon based on recent conversations about making the alerts more useful, I tried not to go overboard in the refactoring.